pkg/solana/txm/txm_internal_test.go

```diff
@@ -767,6 +772,7 @@ func TestTxm_disabled_confirm_timeout_with_retention(t *testing.T) {
 	cfg.Chain.TxRetentionTimeout = relayconfig.MustNewDuration(5 * time.Second)
 	mc := mocks.NewReaderWriter(t)
 	mc.On("GetLatestBlock", mock.Anything).Return(&rpc.GetBlockResult{}, nil).Maybe()
+	mc.On("SlotHeight", mock.Anything).Return(uint64(0), nil).Maybe()
```

Comment:
Do we still need these `SlotHeight` mocks if we've switched to `GetLatestBlock`?

Reply:
Removed!